source/support/update-govwifi-server-certificate.html.erb

```diff
@@ -27,11 +31,61 @@ description: Updating the GovWifi server certificate to keep using the service
 
         <h3 class="govuk-heading-s">Switching to the new certificate on your desktop or laptop</h3>
         <ol class="govuk-list govuk-list--number">
-          <li>If you receive the ‘Authenticating to network “GovWifi”’ pop up message on your desktop or laptop device, select Continue.</li>
+          <li>If you receive the ‘Authenticating to network “GovWifi”’ pop up message on your desktop or laptop device,
+            select Continue.
+          </li>
           <li>Your device will automatically switch to the new certificate and reconnect to GovWifi.</li>
           <li>You can continue using GovWifi as normal.</li>
         </ol>
 
+        <h2 class="govuk-heading-m">What to do if you are unable to connect to GovWifi</h2>
+
+        <p class="govuk-body">If you are unable to connect to GovWifi you can try:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>re-entering your GovWifi username and password</li>
+          <li>restarting your device</li>
+          <li>rejoining your GovWifi network</li>
+          <li>check the left-hand navigation bar to view your device specific instructions</li>
+        </ul>
+
+        <p class="govuk-body">If you are still experiencing problems, contact your IT support desk.</p>
+
+        <h2 class="govuk-heading-m">What to do if you get a Thumbprint/Fingerprint error message</h2>
+        <p class="govuk-body">A Thumbprint, also referred to as a Fingerprint, is an ID that is uniquely calculated from
+          a certificate. The Thumbprint is used to identify and validate the certificate, or as a way to manage
+          certificates in devices. The change of server certificate might cause your device to expect a different
+          Thumbprint.</p>
+        <p class="govuk-body">If you get this error message, you should follow these steps.</p>
+
+        <ol class="govuk-list govuk-list--number">
+          <li>Depending on the device, select to show the Certificate details</li>
+          <li>Check the certificate is called <strong>wifi.service.gov.uk</strong></li>
+          <li>Check the certificate was issued by <strong>GeoTrust RSA CA 2018</strong></li>
+          <li>Check that the Thumbprint matches the latest server certificate from 11
+            November 2019,
+            37:CE:B5:7A:3F:38:EB:7C:66:A2:4E:90:BB:11:71:D7:BC:4D:65:1A
+          </li>
+          <li>Accept the new certificate</li>
+          <li>You can continue using GovWifi as normal</li>
```

Comment:
the punctuation is different than the rest of the pages (other lists use period as the end of each items), maybe we could get our writer to look at these?